Doc/library/binascii.rst

```diff
@@ -158,6 +160,9 @@ The :mod:`binascii` module defines the following functions:
    of hexadecimal digits (which can be upper or lower case), otherwise an
    :exc:`Error` exception is raised.
 
+   Similar functionality (accepting only strings arguments, but more liberal
```

Comment:
string arguments [not strings]

Or even “text string arguments” to be clearer (but other people seem happy assuming “string” excludes byte strings, so maybe that does not matter).